stories/molecules/Teaser.stories.tsx

```diff
@@ -20,14 +20,98 @@ export const WithAllFields: Story = {
         title: titleText,
         subtitle: 'Awsome inc.',
         hasCheckbox: true,
-        timestamp: 'Today',
+        date: 'Today',
         primaryInfo: { text: 'Primary', href: 'https://textkernel.com' },
         secondaryInfo: { text: 'Secondary' },
         tercearyInfo: { text: 'Terceary' },
         sourceInfo: { text: 'Source', icon: <FaLinkedinIn /> },
         isVisited: true,
         matchingIndicatorPercentage: 50,
-        tags: [<Tag bgColor="yellow">First</Tag>, <Tag bgColor="lightblue">Second</Tag>],
+        bottom: (
+            <div style={{ display: 'flex', gap: '8px' }}>
+                <Tag bgColor="yellow" size="small">
+                    First
+                </Tag>
+                <Tag bgColor="lightblue" size="small">
+                    Second
+                </Tag>
+            </div>
+        ),
+    },
```

Comment:
I cannot see any effect on the UI when I set `disabled` or `isSelected`. Also, there is no callback on the checkbox, is there?

Reply:
disabled it was a mistake, that prop should not exist. at least there is no design for disabled. isSelected is the controllable variable of the checkbox status. There is the onChange that is triggered when checkbox status changes.

Reply:
disabled is used, so we need to keep it, otherwise we have a regression. Can you ask Dima to give you a design? Maybe just make the title gray will do...
For example on where we use it, I think it is for some expired jobs in some of our lists